core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -37,26 +43,48 @@ trait RawlsInstrumented extends DefaultInstrumented {
     */
   @implicitNotFound(msg = "Cannot expand instances of type ${A}")
   protected sealed trait Expansion[A] {
-    def makeName(key: String, a: A) = s"$key.${a.toString}"
+    def makeName(key: Option[String], a: A) = s"${getKey(key)}${a.toString}"
```

Comment:
I know this is copy-pasted code, but recently I'm thinking we might want to make `key` non-optional. The reason is that we need to specify a key-value pair when generating stackdriver labels, so we should probably enforce a key in scala code.